src/components/components/ebay-tooltip-base/index.js

```diff
@@ -30,6 +32,11 @@ function init() {
         if (!hostAriaDescribedBy && this.el.parentElement) {
             host.setAttribute('aria-describedby', `${this.el.parentElement.id}-overlay`);
         }
+
+        if (focusableEls[0]) {
```

Comment:
If the component is rerendered with new content then this list of focusable elements can become out of date.

Reply:
Okay. Let me work on that.